### ISSUE TITLE
Create new batch of works accessibility audit 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#3983]
### Descriptive summary

QA Testing for Hyrax
Hyrax Version: 3.0.0.pre.rc1
Hyrax Branch: master
Hyrax Revision: 48ed925
Nurax Version
ENV: production
SHA: 40b47b0d71af1b958e016ebe49212c1740683133
Branch: 2019-09-12-10-00
Updated: 12 Sep 2019 15:00:37

In testing for the Hyrax 3.0 release, the following points are actions that were discovered during a FireFox tab test and axe accessibility audit. Note, only certain issues are being reported at this time.

**Heading levels should only increase by one**
![Screen Shot 2019-09-12 at 6 50 16 PM](https://user-images.githubusercontent.com/9621070/64832293-2e472e80-d58e-11e9-8870-28abf46c2be4.png)

**Aside must not be contained in another landmark**
![Screen Shot 2019-09-12 at 6 50 45 PM](https://user-images.githubusercontent.com/9621070/64832305-4323c200-d58e-11e9-982a-cc7a5e131e71.png)

![Screen Shot 2019-09-12 at 6 51 16 PM](https://user-images.githubusercontent.com/9621070/64832324-59ca1900-d58e-11e9-9dfe-3cc80c9c2602.png)

**Main landmark must not be contained in another landmark**
![Screen Shot 2019-09-12 at 6 52 00 PM](https://user-images.githubusercontent.com/9621070/64832344-6fd7d980-d58e-11e9-89bb-ef7c9c6689dd.png)

**Document must have one main landmark**
![Screen Shot 2019-09-12 at 6 52 39 PM](https://user-images.githubusercontent.com/9621070/64832367-867e3080-d58e-11e9-802f-9c9156856a47.png)

**Ensures landmarks are unique**
![Screen Shot 2019-09-12 at 6 54 06 PM](https://user-images.githubusercontent.com/9621070/64832407-b88f9280-d58e-11e9-99da-874c0dcef54f.png)

![Screen Shot 2019-09-12 at 6 54 26 PM](https://user-images.githubusercontent.com/9621070/64832417-c3e2be00-d58e-11e9-8b43-2952cd84d3b7.png)

![Screen Shot 2019-09-12 at 6 54 47 PM](https://user-images.githubusercontent.com/9621070/64832426-d0671680-d58e-11e9-865b-40f97847b407.png)

**Radio inputs with the same name attribute value must be part of a group**
![Screen Shot 2019-09-12 at 6 55 28 PM](https://user-images.githubusercontent.com/9621070/64832448-e8d73100-d58e-11e9-9b67-85ba7392e5b3.png)

### Steps to reproduce the axe accessibility audit

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Open https://nurax-dev.curationexperts.com/batch_uploads/new?locale=en&payload_concern=GenericWork or click Create batch of works from Works page 
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab
5. Click 'Analyze'

### Tab Test 

In Firefox and Safari on https://nurax-dev.curationexperts.com/dashboard/my/works?locale=en, unable to select the Create batch of words button amongst other links and buttons.

Chrome is working great. 

https://cl.ly/1b659f240c18/download/Screen%252520Recording%2525202019-09-12%252520at%25252006.02%252520PM.mov

When on page https://nurax-dev.curationexperts.com/batch_uploads/new?locale=en&payload_concern=GenericWork#files unable to move to Descriptions, Relationships, or Sharing tabs in Firefox. In Safari, unable to tab through any of the links and buttons.

**Comment by rjkati** [comment]
Tab test passes in Hyrax 3.5.0.

axe plugin is unavailable for Firefox, but reported issues are not present in Hyrax 3.5.0 using WAVE for Firefox.
